hls4ml/model/graph.py

```diff
@@ -514,47 +516,49 @@ def insert_node(self, node, before=None, input_idx=0):
                 new_graph[node.name] = node
 
         self.graph = new_graph
-        self._update_model_outputs()
 
     def remove_node(self, node, rewire=True):
         """Remove a node from a graph.
 
-        By default, this function can connect the outputs of previous node to the input of next one.
-        Note that when removing a leaf node `rewire` should be set to `False`.
+        By default, this function can connect the outputs of previous node to the input of next one. If the removed node has multiple inputs/outputs tensors, an exception is raised.
 
         Args:
             node (Layer): The node to remove
-            rewire (bool, optional): If `True`, connects the outputs of the previous node
-                to the inputs of the next node
+            rewire (bool, optional): Deprecated, no effect
 
         Raises:
-            Exception: If an attempt is made to rewire a leaf node or a node with multiple
-                inputs/outputs.
+            Exception: If an attempt is made to rewire a node with multiple inputs/outputs.
+        """  # noqa: E501
```

Comment:
Minor point:  why the E501? Why not wrap it?

Reply:
Laziness. Though, I did't find it very helpful to wrap long docstrings.
Found a plugin to do it automatically for me thus will change it.